p9/file.go

```diff
@@ -132,6 +132,29 @@ type File interface {
 	// On the server, FSync has a read concurrency guarantee.
 	FSync() error
 
+	// Lock locks the file. The operation as defined in 9P2000.L
+	// is fairly ambitious, being a near-direct mapping to
+	// lockf(2)/fcntl(2)-style locking, but most implementations
+	// use flock(2).
+	//
+	// Arguments are defined by the 9P2000.L standard.
+	//
+	// Pid is the pid on the client.
+	// Locktype is one of read, write, or unlock (resp. 0, 1, or 2).
+	// Flags are to block (0), meaning wait; or reclaim (1),
+	//   which is currently "reserved for future use."
+	// Start and length are the start of the region to use and the size.
+	//   In many implementations, they are ignored and flock(2) is used.
+	// Client is an arbitrary string, also frequently unused.
```

Comment:
"client_id is an additional mechanism for uniquely identifying the lock requester and is set to the nodename by the Linux v9fs client." https://github.com/chaos/diod/blob/master/protocol.md#lock---acquire-or-release-a-posix-record-lock

Reply:
I put that text in, thanks for finding it.